assembl/static2/js/app/containers/search.jsx

```diff
@@ -0,0 +1,266 @@
+/* eslint no-underscore-dangle: 0, react/no-danger: 0 */
+import React from 'react';
+
+import {
+  MenuFilter,
+  // DynamicRangeFilter,
+  SearchBox,
+  Hits,
+  HitsStats,
+  PageSizeSelector,
+  Pagination,
+  ResetFilters,
+  SelectedFilters,
+  SearchkitProvider,
+  SearchkitManager,
+//   NoHits,
+   InitialLoader,
+   Layout, LayoutBody, LayoutResults,
+   SideBar,
+   TopBar,
+   ActionBar, ActionBarRow
+} from 'searchkit';
+import get from 'lodash/get';
+
+import 'searchkit/theming/theme.scss';
+import '../../../css/views/search.scss';
+
+import GlobalFunctions from '../utils/globalFunctions';
+
+const truncate = (text) => {
+  if (!text) {
+    return '';
+  }
+  let modifiedText = text;
+  if (Array.isArray(text)) {
+    // take the first highlight fragment
+    modifiedText = text[0];
+  }
+  modifiedText = modifiedText.substring(0, 400);
+  return modifiedText;
+};
+
+// TODO get the right subject highlight, fr, en in priority, then fallback to und(efined). Same for body obviously.
+// TODO pagination component doesn't seem to work
+// TODO translations fr/en
+// TODO issue with sidebar range filter, it sometime disappears
+// (when there is only 1 result), and the selected range is not kept (maybe because of the date field...)
+
+const PostHit = (props) => {
+  return (
+    <div className={props.bemBlocks.item().mix(props.bemBlocks.container('item'))}>
+      <div className={props.bemBlocks.item('title')}>
+        <a
```

Comment:
Yes, that's the plan. react-router Link pour la v2 et un composant Link qui fait juste un rendu d'un <a href> pour la v1.